pkg/services/postgresql/all_in_one_provision.go

```diff
@@ -146,13 +141,13 @@ func (a *allInOneManager) deployARMTemplate(
 		sdt,
 	)
 	goTemplateParameters := buildGoTemplateParameters(
-		&pp.ServerProvisioningParameters,
+		&pp.DBMSProvisioningParameters,
```

Comment:
Another pervasive change: I've opted for "DBMS" over "Server" wherever it makes sense. The thing we say we'll provision is a DBMS, after all. Where this _doesn't_ change is anywhere where we _are_ specifically talking about a "server." A good example is that even when deprovisioning a DBMS, we use the Azure SDK for Go's `postgresql.ServerClient` (or something like that) to `DeleteServer(...)`. So the deprovisiong step that executes that remains "deprovisionServer".

Reply:
👍